src/fwupd.conf

```diff
@@ -1 +1 @@
-@FWUPD_PLUGINDIR@
+@FWUPD_INTERNAL_LIBDIR@
```

Comment:
I actually realize I don't know why we have this conffile, am I missing something?